release-content/0.16/release-notes/17096_Anamorphic_Bloom.md

```diff
@@ -1,4 +1,12 @@
-<!-- Anamorphic Bloom -->
-<!-- https://github.com/bevyengine/bevy/pull/17096 -->
+Bloom is that soft glow bleeding out from bright areas of light: in the real world, this is caused by your camera (or eyes) getting overwhelmed or an imperfect focus.
+In games, it's a powerful artistic tool for creating everything from cyberpunk neon signs to tastefully glowing windows to juicy geometrically-inspired arcade games.
 
-<!-- TODO -->
+Bevy has had bloom since version 0.9, but we're giving artists another simple lever to tweak: the ability to stretch, squash and otherwise distort the effect by setting the 2-dimensional `scale` parameter on the [`Bloom`] component on your camera.
+
+TODO: add image.
+
+When heavily skewed (usually horizontally), this effect is known as **anamorphic bloom**.
+This effect is associated with a cinematic, futuristic vibe, and emulates the unusual geometry of certain film cameras as they compress a wider image onto narrower film.
```

Comment:
The explanation "why" here is super interesting, but probably not appropriate here. 😆

Reply:
I kind of like it! It's the sort of neat educational fact that folks enjoy reading in our release notes.

Reply:
It's awesome, I love things like these in Bevy release notes 😊

Reply:
I like it too. I think the reason why rendering features exist is super interesting! :)